tests/fixture/tmpnet/node.go

```diff
@@ -89,6 +79,9 @@ type Node struct {
 
 	// Initialized on demand
 	runtime NodeRuntime
+
+	// Intended to be set by the network
```

Comment:
nit such that it links up to the relevant functions setting the network field

```suggestion
	// Intended to be set by [Network.EnsureNodeConfig] or [ReadNodes]
```